docs/Changelog.md

```diff
@@ -18648,6 +18648,11 @@ This version of the operator has been available since version 14 of the default
   
   current_mean = ReduceMean(X, axis=all_except_channel_index)
   current_var =  ReduceVar(X, axis=all_except_channel_index)
+  
+  Notice that ReduceVar refers to the population variance, and it equals to
+  sum(sqrd(x_i - x_avg)) / N
+  where N is the population size (this formula does not use sample size N - 1).
```

Comment:
Don't forget to consider the comment here: https://github.com/onnx/onnx/pull/3406#discussion_r608826460